configuration/wireless/access-point-bridged.md

```diff
@@ -120,6 +120,22 @@ interface br0
 ```
 With this line, interface `br0` will be configured in accordance with the defaults via DHCP. Save the file to complete the IP configuration of the machine.
 
+<a name="wifi-cc-rfkill"></a>
+## Ensure wireless operation
+
+Countries around the world regulate the use of telecommunication radio frequency bands to ensure interference-free operation.
+The Linux OS helps users [comply](https://wireless.wiki.kernel.org/en/developers/regulatory/statement) with these rules by allowing applications to be configured with a two-letter "WiFi country code", e.g. `US` for a computer used in the United States.
+
+In the Raspbian OS, 5 GHz wireless networking is disabled until a WiFi country code has been configured by the user, usually as part of the initial installation process (see wireless configuration pages in this [section](README.md) for details.)
+
+To ensure WiFi radio is not blocked on your Raspberry Pi, execute the following command:
+
+```
+sudo rfkill unblock 0
```

Comment:
Is `0` guaranteed to be the right interface? I don't know how the order they show up in is determined, so I would've used `all` or `wlan` to be safe.

Reply:
cmd changed to rfkill unblock wlan